src/ray/util/process.cc

```diff
@@ -528,6 +532,55 @@ void Process::Kill() {
   }
 }
 
+namespace {
+/// Tracks subprocesses' exit status.
+class ProcessExitStatusTracker {
+ public:
+  static ProcessExitStatusTracker &GetInstance() {
+    static ProcessExitStatusTracker instance;
```

Comment:
Just curious, is it more common to use `static ProcessExitStatusTracker instance;` instead of `static auto* instance = new ProcessExitStatusTracker;` for static local variables in your experience?